Kommunicate/Classes/Views/AwayMessageView.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  AwayMessageView.swift
+//  Kommunicate
+//
+//  Created by Mukesh on 08/01/19.
+//
+
+import Foundation
+import ApplozicSwift
+
+class AwayMessageView: UIView {
```

Comment:
Can we add description to this class and methods? 
It is using core animation and might create problem if we have to change anything, so some snapshot test cases will be helpful.